velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -140,14 +140,29 @@ void toDuckDbFilter(
       }
       break;
     }
+    case common::FilterKind::kBigintValuesUsingHashTable: {
+      auto valuesFilter =
+          static_cast<common::BigintValuesUsingHashTable*>(filter);
+      const auto& values = valuesFilter->values();
+      if (values.size() == 1) {
+        filters.PushFilter(
+            colIdx, constantEqualFilter(makeValue(type, *values.begin())));
+      } else {
+        auto duckFilter = std::make_unique<::duckdb::ConjunctionOrFilter>();
+        for (const auto& value : values) {
+          duckFilter->child_filters.push_back(
+              constantEqualFilter(makeValue(type, value)));
+        }
+        filters.PushFilter(colIdx, std::move(duckFilter));
+      }
+      break;
+    }
 
     case common::FilterKind::kAlwaysFalse:
     case common::FilterKind::kAlwaysTrue:
     case common::FilterKind::kIsNull:
     case common::FilterKind::kIsNotNull:
     case common::FilterKind::kBoolValue:
-    case common::FilterKind::kBigintValuesUsingHashTable:
-    case common::FilterKind::kBigintValuesUsingBitmask:
```

Comment:
Is `kBigintValuesUsingBitmask` implemented?

Reply:
I didn't include the implementation in this PR to keep it small, but accidentally removed it from here. We also found out that Parquet reader doesn't support IN-list filter and requires converting it into a series of ORs.